pkgs/data/themes/flat-remix-gnome/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitHub
+, glib
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "flat-remix-gnome";
+  version = "20210623";
+
+  src = fetchFromGitHub {
+    owner = "daniruiz";
+    repo = pname;
+    rev = version;
+    hash = "sha256-FKsbAvjhXb2ipe3XqACM6OwGYwbBbzvDjsUQYCIQ8NM=";
+  };
+
+  nativeBuildInputs = [ glib ];
+  installPhase = ''
+    # https://github.com/daniruiz/flat-remix-gnome/blob/20210623/Makefile#L50
+    mkdir -p $out/share/gnome-shell/
+    touch $out/share/gnome-shell/gnome-shell-theme.gresource
+
+    make install PREFIX=$out
+    rm $out/share/gnome-shell/gnome-shell-theme.gresource.old
+  '';
```

```suggestion
  preInstall = ''
    # https://github.com/daniruiz/flat-remix-gnome/blob/20210623/Makefile#L50
    mkdir -p $out/share/gnome-shell/
    touch $out/share/gnome-shell/gnome-shell-theme.gresource
  '';
  
  postInstall = ''
    rm $out/share/gnome-shell/gnome-shell-theme.gresource.old
  '';
```

You can get a detail from [default intall phase in generic builder](https://github.com/NixOS/nixpkgs/blob/6ba632c2a442082f353bf2d7028fda11a888d099/pkgs/stdenv/generic/setup.sh#L1094-L1099)